doc/lua_api.txt

```diff
@@ -234,6 +235,40 @@ stripping out the file extension:
 * e.g. `foomod_foothing.png`
 * e.g. `foomod_foothing`
 
+Texture pack directory structure
```

Comment:
It should definitely be a new file. You should add the `Texture pack directory structure` to the texture_overrides.txt file, then rename that file to texture_packs.txt